Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ruby:2.5.1-alpine
+MAINTAINER apply for legal aid team
+
+RUN apk --update add --virtual build-dependencies \
+                                    build-base \
```

Comment:
Could we make the indentation of this and the following lines match the line above? I think it would be easier to read.

Reply:
I've pushed a commit to do this.